README.md

```diff
@@ -2,55 +2,48 @@
 
 ## About
 
-The ROOT system provides a set of modules with all the functionality
-needed to handle and analyze large amounts of data in a very efficient way.
-Having the data defined as a set of objects, specialized storage methods are
-used to get direct access to the separate attributes of the selected objects,
-without having to touch the bulk of the data. Included are histograming
-methods in an arbitrary number of dimensions, curve fitting, function
-evaluation, minimization, graphics and visualization classes to allow
-the easy setup of an analysis system that can query and process the data
+ROOT is a unified software package for the storage, processing, visualization 
+and analysis of scientific data. It is reliable, performant and well supported,
+easy to use and obtain, and strives to minimize the computing resources needed 
+to achieve scientific results.
+
+ROOT is a comprehensive toolbox to process and analyze data, from its acquisition, 
+to its processing, analysis and final visualization in form of highly customizable, 
+publication-ready plots.
+First of all, ROOT provides a very efficient storage system for data models, 
+that demonstrated to scale at the Large Hadron Collider experiments: Exabytes 
+of scientific data are written in columnar ROOT format.
+Moreover, ROOT comes with histogramming capabilities in an arbitrary number of 
+dimensions, curve fitting, statistical modelling, minimization, to allow
+the easy setup of a data analysis system that can query and process the data
 interactively or in batch mode, as well as a general parallel processing
-framework, PROOF, that can considerably speed up an analysis.
+framework, RDataFrame, that can considerably speed up an analysis, taking 
+full advantage of multi-core and distributed systems.
 
-Thanks to the built-in C++ interpreter cling, the command, the
-scripting and the programming language are all C++. The interpreter
-allows for fast prototyping of the macros since it removes the time
-consuming compile/link cycle. It also provides a good environment to
-learn C++. If more performance is needed the interactively developed
-macros can be compiled using a C++ compiler via a machine independent
-transparent compiler interface called ACliC.
+ROOT is written in C++, and, thanks to PyROOT, provides a powerful Python interface 
```

Comment:
Can we ack `cppyy` as it relates to PyROOT in the same way as Cling to ROOT?

Reply:
make a proposal?

Reply:
Something like:

ROOT is performance critical software written in C++ and enables rapid prototyping powered by a unique C++ compliant interpreter called Cling. Cling enables efficient C++ type introspection which is a building block of flexible rapid prototyping in C++ and efficient automatic bindings with Python. The PyROOT component offers a powerful Python interface leveraging the [cppyy technology](https://cppinterop.readthedocs.io/). Cppyy uses Cling to connect to Python's runtime providing efficient, on-demand C++/Python interoperability.